addon/components/o-s-s/attributes-panel.stories.js

```diff
@@ -46,6 +46,16 @@ export default {
       },
       control: { type: 'boolean' }
     },
+    initialMode: {
+      description: 'Automatically enable edit mode',
```

Comment:
Allows setting the initial mode of the component. Can be either "edit" or "view".